dm/pkg/schema/tracker.go

```diff
@@ -201,6 +202,8 @@ func NewTracker(ctx context.Context, task string, sessionCfg map[string]string,
 			return nil, err
 		}
 	}
+	// skip DDL test https://github.com/pingcap/tidb/pull/33079
+	se.SetValue(sessionctx.QueryString, "skip")
```

Comment:
just a question： do we need set this var in every func? can we just set once when `tracker` init

Reply:
tracker.Exec will write this field, and later DDL API will see outdated value.